src/cmd/cmdgazebo.rb.in

```diff
@@ -118,53 +118,53 @@ COMMANDS = { 'gazebo' =>
+  "                               GZ_SIM_PHYSICS_ENGINE_PATH.                  \n"\
   "\n"\
-  "  --render-engine [arg]        Ignition Rendering engine plugin to load for     \n"\
+  "  --render-engine [arg]        Gazebo Rendering engine plugin to load for     \n"\
   "                               both the server and the GUI. Gazebo will use     \n"\
   "                               OGRE2 by default. (ogre2)                        \n"\
   "                               Make sure custom plugins are in                  \n"\
-  "                               IGN_GAZEBO_RENDER_ENGINE_PATH.                   \n"\
+  "                               GZ_SIM_RENDER_ENGINE_PATH.                   \n"\
   "\n"\
-  "  --render-engine-gui [arg]    Ignition Rendering engine plugin to load for     \n"\
+  "  --render-engine-gui [arg]    Gazebo Rendering engine plugin to load for     \n"\
   "                               the GUI. Gazebo will use OGRE2 by default.       \n"\
   "                               (ogre2)                                          \n"\
   "                               Make sure custom plugins are in                  \n"\
-  "                               IGN_GAZEBO_RENDER_ENGINE_PATH.                   \n"\
+  "                               GZ_SIM_RENDER_ENGINE_PATH.                   \n"\
   "\n"\
-  "  --render-engine-server [arg] Ignition Rendering engine plugin to load for     \n"\
+  "  --render-engine-server [arg] Gazebo Rendering engine plugin to load for     \n"\
   "                               the server. Gazebo will use OGRE2 by default.    \n"\
   "                               (ogre2)                                          \n"\
   "                               Make sure custom plugins are in                  \n"\
-  "                               IGN_GAZEBO_RENDER_ENGINE_PATH.                   \n"\
+  "                               GZ_SIM_RENDER_ENGINE_PATH.                   \n"\
   "\n"\
   "  --version                    Print Gazebo version information.                \n"\
   "\n"\
   "  -z [arg]                     Update rate in Hertz.                            \n"\
   "\n"+
   COMMON_OPTIONS + "\n\n" +
   "Environment variables:                                                  \n"\
-  "  IGN_GAZEBO_RESOURCE_PATH    Colon separated paths used to locate      \n"\
+  "  GZ_SIM_RESOURCE_PATH    Colon separated paths used to locate      \n"\
   " resources such as worlds and models.                                 \n\n"\
-  "  IGN_GAZEBO_SYSTEM_PLUGIN_PATH    Colon separated paths used to        \n"\
+  "  GZ_SIM_SYSTEM_PLUGIN_PATH    Colon separated paths used to        \n"\
   " locate system plugins.                                               \n\n"\
-  "  IGN_GAZEBO_SERVER_CONFIG_PATH    Path to server configuration file. \n\n"\
+  "  GZ_SIM_SERVER_CONFIG_PATH    Path to server configuration file. \n\n"\
   "  IGN_GUI_PLUGIN_PATH    Colon separated paths used to locate GUI       \n"\
```

Comment:
```suggestion
  "  GZ_GUI_PLUGIN_PATH    Colon separated paths used to locate GUI       \n"\
```

Reply:
I took your advice and reverted the env vars for `gz-sim` specifically.
https://github.com/gazebosim/gz-sim/pull/1496/commits/39c3200f39e73f66d4e2d23d1753e1531012e1b2